### PR TITLE
Add translations nl

### PR DESCRIPTION
**Pull request by pablothedude**
https://www.pivotaltracker.com/story/show/176891048

### CODE REVIEW

File: src/Infrastructure/Locale/LocaleSubscriber.php [hunk 1]
```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Copyright 2010 SURFnet B.V.
```
Comment (MKodde) [inline src/Infrastructure/Locale/LocaleSubscriber.php:6]: Please set this to either the year this product was written, or use the current year. Also applies for the associated test

File: translations/messages.nl.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+readid.authorization.title: 'Identiteitscontrole'
+readid.authorization.text: 'Onherstelbare error'
```
Comment (MKodde) [inline translations/messages.nl.yaml]: Maybe 'fout' is more mainstream (and iirc is used in the other openconext applications)

File: src/Infrastructure/Locale/LocaleSubscriber.php [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+ * limitations under the License.
+ */
+
+namespace StepupReadId\Infrastructure\Locale;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\RequestEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+class LocaleSubscriber implements EventSubscriberInterface
```
Comment (MKodde) [inline src/Infrastructure/Locale/LocaleSubscriber.php:27]: Code looks familiar, is this copied from the StepupBundle?

**Review by MKodde — CHANGES_REQUESTED** [review]

File: src/Infrastructure/Locale/LocaleSubscriber.php (hunk 1 above)
Comment (pablothedude) [reply src/Infrastructure/Locale/LocaleSubscriber.php:6]: I'll  change this all at once in https://github.com/OpenConext/Stepup-ReadID/pull/27

File: src/Infrastructure/Locale/LocaleSubscriber.php (hunk 3 above)
Comment (pablothedude) [reply src/Infrastructure/Locale/LocaleSubscriber.php:27]: No

**Review by MKodde — APPROVED** [review]

**Pull request merged**